scripts/update_torch_wheels.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# Must be YYYYMMDD format
+DATE=$1
+TORCH_WHEEL="gs://pytorch-tpu-releases/wheels/torch-1.1.0a0+stable-cp35-cp35m-linux_x86_64.whl"
+TORCH_XLA_WHEEL="gs://pytorch-tpu-releases/wheels/torch_xla-0.1+nightly-cp35-cp35m-linux_x86_64.whl"
+if [ ! -z "$DATE" ]; then
+  printf 'Which of the following torch wheels would you link to install?\n'
+  gsutil ls "gs://pytorch-tpu-releases/wheels/torch-${DATE}-*"
+  printf '\n(copy paste and enter one of above): '
+  read TORCH_WHEEL
+  printf '\nWhich of the following torch_xla wheels would you link to install?\n'
+  gsutil ls "gs://pytorch-tpu-releases/wheels/torch_xla-${DATE}-*"
+  printf '\n(copy paste and enter one of above): '
+  read TORCH_XLA_WHEEL
+fi
+gsutil cp "$TORCH_WHEEL" /tmp/.
```

Comment:
I'd remove the dot after /tmp, and cleanup the files after install. /tmp is RAM ;)

Reply:
Thanks! Comments addressed in: https://github.com/pytorch/xla/pull/681